hugegraph-server/hugegraph-dist/src/assembly/travis/run-unit-test.sh

```
@@ -19,6 +19,9 @@ set -ev
 
 BACKEND=$1
 
+# install rocksdb
+source $TRAVIS_DIR/install-rocksdb.sh
```

Shellcheck (0.11.0)

[info] Not following: ./install-rocksdb.sh was not specified as input (see shellcheck -x).

(SC1091)